schemas/TransferStateReceipt.json

```diff
@@ -24,7 +46,19 @@
           }]
         }
       }
+    },
+    "condition_state": {
+      "description": "Transfer state",
```

Comment:
I'm actually not sure what `condition_state` is. Do we have a list of values for this field?